.github/workflows/codeql.yml

```diff
@@ -0,0 +1,38 @@
+name: CodeQL
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  # schedule:
+  #   - cron: '0 10 * * *'
+
+jobs:
+  analyze:
+    name: Analyze with CodeQL
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]

```

Comment:
Was wondering if they support TypeScript, but seems like TypeScript == JavaScript to CodeQL 🙂 
https://github.com/github/codeql-action/blob/c6b33b9ec15bc742ec514cc62a08ae2e608aae0a/src/count-loc.ts#L20

I wonder if we should specify `typescript`, in case they change it to use an independent parser in the future?

Reply:
Seems like it works just fine! : ) 
Keeping TS 😋